core/src/main/java/org/apache/calcite/rel/metadata/RelMdPredicates.java

```diff
@@ -194,15 +194,11 @@ public RelOptPredicateList getPredicates(Project project,
     final List<RexNode> projectPullUpPredicates = new ArrayList<>();
 
     ImmutableBitSet.Builder columnsMappedBuilder = ImmutableBitSet.builder();
-    Mapping m =
-        Mappings.create(MappingType.PARTIAL_FUNCTION,
-            input.getRowType().getFieldCount(),
-            project.getRowType().getFieldCount());
-
+    Map<Integer, BitSet> equivalence = new HashMap<>();
```

Comment:
can you add a comment describing what this map does, including the keys and values?

Reply:
Done!